rpm/tarantool.spec

```diff
@@ -208,7 +222,12 @@ make %{?_smp_mflags}
 rm -rf %{buildroot}%{_datarootdir}/doc/tarantool/
 
 %check
+%if 0%{?fedora} >= 36
+# Workaround to make app/crypto.test.lua and app/digest.test.lua tests work.
```

Comment:
I would add some link(s) here, so a developer can re-check, whether the problem is resolved and whether the workaround can be removed, in the future.

Reply:
fixed